test/color/test_ycbcr.py

```diff
@@ -0,0 +1,160 @@
+import pytest
+
+import kornia
+import kornia.testing as utils  # test utils
+
+import cv2
+import numpy as np
+
+import torch
+from torch.autograd import gradcheck
+from torch.testing import assert_allclose
+
+
+class TestRgbToYcbcr:
+
+    def test_rgb_to_ycbcr(self):
+
+        data = torch.rand(3, 5, 5)
+
+        # OpenCV
+        data_cv = data.numpy().transpose(1, 2, 0).copy()
+        expected = cv2.cvtColor(data_cv, cv2.COLOR_RGB2YCrCb)
```

Comment:
not sure here what makes sense to do, see @priba latest PR. He hard-coded expected values